File: MY PROGRAMS/LB/JAVA/NUMBER/a3q5.java
/*	5. Accept on character from user and check whether that character is vowel (a,e,i,o,u) or not.

	Input:	E
	Output:	true

	Input:	d
	Output:	false		*/




import java.util.*;

class Assignment3
{
	public boolean Boolean(char ch)
	{
		if((ch=='a') || (ch=='e') || (ch=='i') || (ch=='o') || (ch=='u') || (ch=='A') || (ch=='E') || (ch=='I') || (ch=='O') || (ch=='U'))
		{
			return(true);
		}
		else
		{
			return(false);
		}
	}
}

class a3q5
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		char cvalue='\0';
		boolean bret=false;

		System.out.println("Enter the character: ");
		cvalue=sobj.next().charAt(0);

		Assignment3 aobj=new Assignment3();

		bret=aobj.Boolean(cvalue);

		if(bret==true)
		{
			System.out.println("YES,The given character is vowel.");
		}
		else	
		{
			System.out.println("NO,The given character is not vowel.");
		}
	}
}

File: MY PROGRAMS/LOGIC BUILDING-13 ASSIGNMENTS/a3q5.c
/* Accept on character from user and check whether that character is vowel (a,e,i,o,u) or not.

Input:E
Output:YES

Input:d
Output:No 

*/




#include<stdio.h>

#define yes 1
#define no 0

typedef int BOOL;

BOOL Vowel(char ch)
{
	if((ch=='a') || (ch=='e') || (ch=='i') || (ch=='o') || (ch=='u') || (ch=='A') || (ch=='E') || (ch=='I') || (ch=='O') || (ch=='U'))
	{
		return(yes);
	}
	else	
	{
		return(no);
	}
}

int main()
{
	char cvalue='\0';
	BOOL bret=no;

	printf("Enter the character:");
	scanf("%c",&cvalue);

	bret=Vowel(cvalue);

	if(bret==yes)
	{
		printf("It is a vowel.");
	}
	else
	{
		printf("It is not a vowel.");
	}

	return(0);
}
